astro.config.mjs

```diff
@@ -45,8 +45,11 @@ export default defineConfig({
   integrations: [
     starlight({
       title: "Nemo Rule Engine",
+      logo: {
+        dark: "./src/assets/nemo-logo-rusty-bright-nomargin.svg",
+        light: "./src/assets/nemo-logo-rusty-nomargin.svg",
+      },
       customCss: ["./src/tailwind.css"],
-      favicon: "/favicon.svg",
```

Comment:
Don't drop the favicon here, this will get build by the `vite` plugin

Reply:
weird, I thought I deleted that. must have slipped back in